scipy/stats/_stats_mstats_common.py

```diff
@@ -66,28 +71,33 @@ def linregress(x, y=None):
 
     Perform the linear regression:
 
-    >>> slope, intercept, r_value, p_value, std_err = stats.linregress(x, y)
-    >>> print("slope: %f    intercept: %f" % (slope, intercept))
-    slope: 1.944864    intercept: 0.268578
+    >>> res = stats.linregress(x, y)
 
-    To get coefficient of determination (R-squared):
+    Coefficient of determination (R-squared):
 
-    >>> print("R-squared: %f" % r_value**2)
+    >>> print(f"R-squared: {res.rvalue**2:.6f}")
     R-squared: 0.735498
 
     Plot the data along with the fitted line:
 
     >>> plt.plot(x, y, 'o', label='original data')
-    >>> plt.plot(x, intercept + slope*x, 'r', label='fitted line')
+    >>> plt.plot(x, res.intercept + res.slope*x, 'r', label='fitted line')
     >>> plt.legend()
     >>> plt.show()
 
-    Example for the case where only x is provided as a 2x2 array:
+    Calculate 95% confidence interval on slope and intercept:
+
+    >>> # Two-sided inverse Students t-distribution
+    >>> # p - probability, df - degrees of freedom
+    >>> from scipy.stats import t
+    >>> tinv = lambda p,df: abs(t.ppf(p/2,df))
```

Comment:
Stick to [PEP-8](https://www.python.org/dev/peps/pep-0008/) in the sample code; in this case, we need a space after the commas.
```suggestion
    >>> tinv = lambda p, df: abs(t.ppf(p/2, df))
```